### ISSUE TITLE
Brick Feature Request: Local installation and testing new bricks.

### ISSUE THREAD

**Issue by rnlambert7** [Gottwik/Enduro#210]
TL:DR; Enduro needs more options when installing Bricks. A user should be able to easily build their own locally and then install them for testing. There is far too much overhead for testing a new one out now.

Long Form)
Before Bricks were finalized my team began building an app using the Pell WYSIWYG that a contributor had come up with. However after an update, the custom types ability was removed and put into bricks. The old style of creating custom_types that I've seen others do is no longer usable. So now I am building a Brick using Pell since it was previously already being used.

From following the installation code (There isn't clear documentation on what happens when you install a brick), it appears that I need to create my brick, and load it up on npm in order to install it, and _then_ I can test. This isn't a very welcoming way to develop our own. It would be fantastic if I had either A) Instructions for where all to drop brick code in without using the CLI command, or B) a brick local install, wherein I could enter the directory on my machine and it would install the brick.
